.github/workflows/reviewpad.yml

```diff
@@ -22,6 +22,12 @@ jobs:
   reviewpad_job:
     runs-on: ubuntu-latest
     name: Reviewpad
+    # Don't run reviewpad on automated PRs
+    if: |
+      (github.event_name != "pull_request_target") ||
+      ((github.event.pull_request.title != "Merge new release into main") &&
+      (github.event.pull_request.title != "Merge testimonial into main") &&
+      (github.event.pull_request.title != "Merge formatting into main"))
```

Comment:
I wonder if we could prepend all the automated pull requests with something like `AUTOMATED: ...` then we can just check for that - this could come as a future improvement, it is not blocking this PR just was an idea that came to mind

Reply:
I thinking changing 1 file instead of 4 is better in the short term. If this works as expected, I will come back to it.